frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -34,26 +34,28 @@ We need the following collection metadata (i.e. details associated with your pub
   - Publication/preprint DOI: can be added later
   - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
 
-Datasets need the following information added to a single h5ad (AnnData 0.7) format file per CELLxGENE visualization:
+Datasets need the following information added to a single h5ad (AnnData 0.8) format file per CELLxGENE visualization:
 
 - **Dataset-level metadata in uns**:
-  - schema_version: `2.0.0`
-  - title: title of the individual dataset visualization
-  - optional: batch_condition: list of obs fields that define that “batches” that a normalization or integration algorithm should be aware of
+  - schema_version: `3.0.0`
+  - title: title of the individual dataset
+  - optional: batch_condition: list of obs fields that define “batches” that a normalization or integration algorithm should be aware of
 - **Data in .X and raw.X**:
   - raw counts are required
   - normalized counts are strongly recommended
   - raw counts should be in raw.X if normalized counts are in .X
   - if there is no normalized matrix, raw counts should be in .X
-- **Cell metadata in obs (the values MUST be the most specific specific term available from the specified ontology)**:
+- **Cell metadata in obs (for ontology term IDs, the values MUST be the most specific term available from the specified ontology)**:
```

Comment:
@jahilton - was this list created in some priority order or could it be sorted? It seems random. I sort in the schema.

Reply:
It is random. When I present this non-randomly, I use something like this logical order of high-to-low level biology to technical:

- organism
- donor_id
- dev_stage
- sex
- ethnicity
- disease
- tissue
- cell_type
- assay
- suspension_type

Reply:
@brianraymor should we update this list to follow the ordering jason outlined above?

Reply:
I think that would best since Jason can reuse this information for _first contact_.